input/input.css

```diff
@@ -0,0 +1,131 @@
+@import url(/style.css);
+
+.input-intro {
+  font-size: 1.25rem;
+  padding: 1rem 1rem;
```

Comment:
you can use padding:1rem instead of padding: 1rem 1rem

Reply:
okay got it.